tests/docs/additional-language-support.md

```diff
@@ -0,0 +1,5 @@
+This is an example REST response
+
+```json 
+{body: {results: [spam, eggs]}, errors: []} 
```

Comment:
Isn't this invalid JSON? Shouldn't it be:

```
{body: {results: ["spam", "eggs"]}, errors: []} 
```

Reply:
Yes  of course. This is exactly why documentation needs testing 🙃